### PR TITLE
Bugfix

### LINKED ISSUE #6: Make naming consistent with other chat bot projects
The other chat bot projects spell chat bot as 2 words. This repo spells chat bot as 1 word.

- [x] Rename this repo to `chat-bot-training-data`
- [x] Delete existing deployment
- [x] Update `APP` variable in `Makefile`
- [x] Deploy with new name

### LINKED ISSUE #7: Disable metrics in deployment
B/c of the new http-service-chart updates the `http.metricsEnabled` key in `deploy/values.yaml` must be set to false. Currently an alert is firing b/c the existing deployment created a metrics service and it is down.

- [x] Set `http.metricsEnabled` to `false`
- [x] Delete the metrics service created by the deployment:
    ```
    oc delete svc prod-chatbot-training-data-metrics
    ```

### PR DESCRIPTION
**Pull request by PratikMahajan**
fixes #6 
fixed #7

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -98,7 +98,7 @@ open-proxy:
 DB_DATA_DIR ?= container-data/db
 DB_CONTAINER_NAME ?= prod-kscout-bot-api-db
 DB_USER ?= prod-kscout-bot-api
-DB_PASSWORD ?= secretpassword
+DB_PASSWORD ?= ${DB_PASS}
```
Comment (catvec) [inline Makefile]: What is the reason behind this change? Couldn't users just set `DB_PASSWORD` before? Now they have to set `DB_PASS` to set `DB_PASSWORD`?

**Review by catvec — CHANGES_REQUESTED** [review]
Fixes to #6 & #7 look good. However there is something weird going on with the `DB_PASSWORD` var `Makefile` that I would like to investigate further.

File: Makefile [hunk 2]
```diff
@@ -98,7 +98,6 @@ open-proxy:
 DB_DATA_DIR ?= container-data/db
 DB_CONTAINER_NAME ?= prod-kscout-bot-api-db
 DB_USER ?= prod-kscout-bot-api
-DB_PASSWORD ?= secretpassword
```
Comment (catvec) [inline Makefile:101]: Why was `DB_PASSWORD` removed? It is still used in the Makefile? 

```
DB_PASSWORD ?= secretpassword
```
Makes it so the `DB_PASSWORD` variable will be set to `secretpassword` by default.

Also can we move these variables to the top of the Makefile where the other variables are?

**Review by catvec — CHANGES_REQUESTED** [review]
Still confused about the `DB_PASSWORD` var situation.

Why are we removing it?

File: Makefile (hunk 2 above)
Comment (PratikMahajan) [reply Makefile:101]: Was thinking of keeping password only ENV and not visible in Makefile

Comment (catvec) [reply Makefile:101]: Why? This will just make development harder for people. Because they will have to manually set this variable.

The variable is set in the Makefile isn't used in production. It is used only for local development. Setting a default value for `DB_PASSWORD` makes it so most people can just type `make db` and `make db-cli` with little effort.

- [x] Add `DB_PASSWORD` variable back with default value

Comment (PratikMahajan) [reply Makefile:101]: #GetTheDBPasswordBack

Comment (catvec) [reply Makefile:101]: :joy:

Comment (catvec) [reply Makefile:101]: DB Password Gate

**Review by catvec — APPROVED** [review]
lgtm

File: Makefile (hunk 2 above)
Comment (PratikMahajan) [reply Makefile:101]: 😂
